nix/status-go/build-desktop-status-go.nix

```
@@ -14,19 +14,25 @@ let
     inherit buildGoPackage go xcodeWrapper utils;
   };
 
+  # Remove desktop-only arguments from args
   args = removeAttrs args' [
-    "goBuildFlags"
-    "goBuildLdFlags"
-    "outputFileName"
-    "hostSystem"
-  ]; # Remove desktop-only arguments from args
+    "goBuildFlags" "outputFileName" "hostSystem" "goBuildLdFlags"
```

Why not keep both flags-related names together?
```suggestion
    "goBuildFlags" "goBuildLdFlags" "outputFileName" "hostSystem"
```

---

Sure.